controllers/snapshot/snapshot_adapter.go

```diff
@@ -379,15 +379,15 @@ func (a *Adapter) createIntegrationPipelineRun(application *applicationapiv1alph
 	return nil
 }
 
-// createSnapshotEnvironmentBindingForSnapshot creates and returns a new snapshotEnvironmentBinding
+// newSnapshotEnvironmentBindingForSnapshot creates and returns a new snapshotEnvironmentBinding
 // for the given application, environment, snapshot, and components.
 // If it's not possible to create it and set the application as the owner, an error will be returned
-func (a *Adapter) createSnapshotEnvironmentBindingForSnapshot(application *applicationapiv1alpha1.Application,
+func (a *Adapter) newSnapshotEnvironmentBindingForSnapshot(application *applicationapiv1alpha1.Application,
```

Comment:
Since this functions [does create the resource on the cluster](https://github.com/redhat-appstudio/integration-service/pull/80/files#diff-8e718c6bf06f754db7e9e94c6f74b5c6704941dff95d637289e3c4cc194c95a0R400), I think it makes sense to keep the name with `create*` prefix here.

Reply:
Glad to see this is resolved